### DISCUSSION TITLE
Adding new Backlit effect

### DISCUSSION THREAD

**Discussion by speeedlover** [seerge/g-helper#3368] [Ideas]
Can you add an extra backlit option.... in which keyboard backlight visually displays frequently used keys. Like when you press any key from the keyboard, the key and other keys surrounding it glows up. I have attached a video for better understanding.

https://github.com/user-attachments/assets/bb3b9b63-334d-4ef0-a959-46e39a28a5ad

 I thought the "heatmap" option that is already added in ghelper does the same what I am telling you to add, but later I found out that it only changes into 3 colors depending on the cpu temp.

**Comment by seerge** [comment]
@speeedlover hello - this can be very difficult to do, cause G-Helper will have to listen to all your keypresses. And it doesn't do that atm. More over, lots of people may not like that.

Also you laptop need to support per-key backlight controls, which is the case only for ROG Strix Scar models.
